SkillFeatureProjectMars/StepDefinitions/EditSkillStepDefinitions.cs

```diff
@@ -0,0 +1,34 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using SkillFeature_Mars.Drivers;
+using SkillFeature_Mars.Pages;
+using System;
+using TechTalk.SpecFlow;
+
+namespace SkillFeature_Mars.StepDefinitions
+{
+    [Binding]
+    public class EditSkillStepDefinitions : CommonDriver
+    {
+        EditSkillPage editSkillPageObj = new EditSkillPage();
```

Comment:
make use of constructors
Please see
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors